Update `tissue_type` field with new CDE

jsunny23 opened the pull request:
This PR updates the `tissue_type` sample field with a new CDE that is not retired. The values of the new CDE are a bit different from the previous one.

Before submitting this PR, please make sure:

- [x] You have added a few sentences describing the PR here.
- [x] You have added yourself or the appropriate individual as the assignee.
- [x] You have added the relevant groups/individuals to the reviewers.
- [x] Your commit messages conform to the [Conventional
      Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard.
- [x] You have updated the README or other documentation to account for these
      changes (when appropriate).
- [x] You have added a line describing the change in the `CHANGELOG.md` under
      `[Unreleased]`.

<!--

If you are adding new metadata elements, please uncomment this section and
complete the checklist as well:

- [ ] I have added my field definition to the appropriate
  `get_field_descriptions()` method. For example, if you add a field to
  subjects, you should include it in the `get_field_descriptions()` method at
  `crates/ccdi_models/src/metadata/field/description/harmonized/subject.rs`.
- [ ] I have confirmed that my field shows up in the relevant
  `/metadata/fields/<entity>` endpoint. For example. if you add a field to
  subjects, it should show up in the fields listed in the output of the
  `/metadata/fields/subject` endpoint.
- [ ] I have confirmed that my field filters correctly when filtered from the
  root endpoint (`/subject`, `/sample`, etc). For example, if you add the
  `anatomical_sites` field to the sample endpoint, make sure that visiting
  `http://localhost:8000/sample?anatomical_sites=foobar` works.
- [ ] I have confirmed that my field shows up in the relevant wiki generation
  command. For example. if you add a field to subjects, it should show up in the
  `cargo run --release wiki subject` output.

-->

e-t-k commented on crates/ccdi-cde/src/v2/sample/tissue_type.rs:
Because the new CDE is `v1.0.0` while the old one was `v2.0.0` , I unfortunately think this needs to be moved to the [v1 directory](https://github.com/CBIIT/ccdi-federation-api/tree/main/crates/ccdi-cde/src/v1), and references updated. For example https://github.com/CBIIT/ccdi-federation-api/blob/main/crates/ccdi-models/src/sample/metadata/builder.rs#L196 but there are probably more.

e-t-k reviewed (commented):
Other than the 1.0 vs 2.0 issue, looks great. Thank you!

Also, I don't want to forget that [the wiki ](https://github.com/CBIIT/ccdi-federation-api/wiki/Sample-Metadata-Fields#tissue_type) will need to change when this update goes live; I'm not sure if that is auto-generated somehow or if we need to change it by hand.

jsunny23 commented:
I've moved the CDE to the correct directory and updated the references to the new location.

Thanks for bringing up the wiki, I will update that in conjunction with merging this PR. Just to be explicit, I'm going to run `cargo run --release --bin ccdi-spec wiki sample` and then paste that into the sample wiki page when I merge.

e-t-k reviewed (approved):
Looks good to me!
